backend/src/services/user-details/user-details.service.ts

```diff
@@ -34,6 +34,15 @@ class UserDetails {
     userId: number,
     userDetailsUpdateInfoRequestDto: UserDetailsUpdateInfoRequestDto,
   ): Promise<UserDetailsResponseDto | null> {
+    const { telegramUsername } = userDetailsUpdateInfoRequestDto;
+
+    if (telegramUsername === '') {
+      userDetailsUpdateInfoRequestDto = {
+        ...userDetailsUpdateInfoRequestDto,
+        telegramUsername: null,
+      };
+    }
```

Comment:
```suggestion
   const hasTelegram = Boolean(userDetailsUpdateInfoRequestDto. telegramUsername);
   
   const userDetailsUpdateInfoRequestDto = {
     ...userDetailsUpdateInfoRequestDto,
     telegramUsername: hasTelegram ? telegramUsername : null
   }
    if (telegramUsername === '') {
      userDetailsUpdateInfoRequestDto = {
        ...userDetailsUpdateInfoRequestDto,
        telegramUsername: null,
      };
    }
```

Reply:
But from frontend we will always receive string (or empty string) for telegram, so why do we need to check it?